Environments/SharedServices/pipeline.yml

```diff
@@ -623,20 +623,19 @@ stages:
         ScriptPath: 'Orchestration/OrchestrationService/ValidationResourceGroupSetup.ps1'
         ScriptArguments: '-TearDownResourceGroup'
         azurePowerShellVersion: 'LatestVersion'
-- stage: Deploy
+- stage: ApplyGovernance
   jobs:
   - job: DiagnosticStorageAccount
-    timeoutInMinutes: 0
     pool:
       name: 'vdc-self-hosted'
     steps:
     - task: AzurePowerShell@4
       displayName: "Diagnostic Storage Account" 
       inputs:
-        azureSubscription: 'vdc2-hub'
+        azureSubscription: 'vdc2-spoke1'
```

Comment:
Yes, just updated it, the reason the deployment created the resources in the right subscription is because we don't rely on the subscription specified here, but is fixed.